src/security/WSSecurityCert.ts

```diff
@@ -111,15 +112,19 @@ export class WSSecurityCert implements ISecurity {
       key: privatePEM,
       passphrase: password,
     };
+    this.keyInfoId = `KI-${generateId()}`;
     this.x509Id = `x509-${generateId()}`;
     this.hasTimeStamp = typeof options.hasTimeStamp === 'undefined' ? true : !!options.hasTimeStamp;
     this.signatureTransformations = Array.isArray(options.signatureTransformations) ? options.signatureTransformations
       : ['http://www.w3.org/2000/09/xmldsig#enveloped-signature', 'http://www.w3.org/2001/10/xml-exc-c14n#'];
 
     this.signer.getKeyInfo = (key) => {
```

Comment:
Looks like I missed this when was migrating to the latest XML-Crypto. I think you can set `getKeyInfoContent` and it will add `KeyInfo`. 

```suggestion
    this.signer.getKeyInfoContent = (key) => {
```

Reply:
used `getKeyInfoContent` from `xml-crypto` as requested.